sell-loot.lic

```diff
@@ -19,24 +19,31 @@ class SellLoot
 
     settings = get_settings
     town_data = get_data('town')
-    @actual_fucking_hometown = settings.hometown
+    @character_hometown = settings.hometown
     @hometown = town_data[settings.hometown]
     @bankbot_name = settings.bankbot_name
     @bankbot_deposit_threshold = settings.bankbot_deposit_threshold
     @bankbot_room_id = settings.bankbot_room_id
-
+    bankbot_skip_bank = settings.bankbot_skip_bank
+    @local_currency = town_data[@character_hometown]['currency']
+    
     sell_gems("#{settings.gem_pouch_adjective} pouch") if settings.sell_loot_pouch
-
     check_spare_pouch(settings.spare_gem_pouch_container, settings.gem_pouch_adjective) if settings.spare_gem_pouch_container
 
     sell_bundle if settings.sell_loot_bundle
 
-    return if settings.sell_loot_skip_bank
-    exchange_coins
+    @keep_coppers = parse(settings.sell_loot_money_on_hand.split(' ')) + @bankbot_deposit_threshold
 
-    keep_copper = parse(settings.sell_loot_money_on_hand.split(' '))
-    deposit_coins(keep_copper)
-    check_bankbot(settings.hometown, keep_copper)
+    if settings.sell_loot_skip_bank && settings.bankbot_skip_bank
+      ['lirums', 'kronars', 'dokoras'].each{|currency| 
```

Comment:
I can't say I care much about three super common pieces of information that everyone knows, but it wouldn't be hard to move it into common-money.